tests/transports-integrations/Makefile

```
@@ -0,0 +1,117 @@
+# Bifrost Python E2E Test Makefile
+# Provides convenient commands for running tests
+
+.PHONY: help install test test-all test-parallel test-verbose clean lint format check-env
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)